doc/nrf/known_issues.rst

```diff
@@ -723,15 +754,15 @@ KRKNWK-6073: Potential delay during FOTA
 Matter (Project CHIP)
 =====================
 
```

Comment:
Could we add the following here or should I create a separate PR?
```suggestion

.. rst-class:: v1-9-0

KRKNWK-12950: Android CHIPTool opens the commissioning window using an incorrect PIN code
  Android CHIPTool uses a random instead of a user-provided PIN code to open the commissioning window on a Matter device.

```